11-Reset/readme.md

```diff
@@ -0,0 +1,67 @@
+git-reset
+=========
+
+> Reset current HEAD to the specified state
+
+Git reset has three forms. &lt;tree-ish&gt; defaults to HEAD in all of them.
+
+git reset [-q] [<tree-ish>] [--] <paths>...
+-------------------------------------------
+
+> Can be thought of as opposite for `git-add`.
+
+Resets index entries for all &lt;paths&gt; to their state at &lt;tree-ish&gt;. Important part here is that this form of `git-reset` doesn't affect the working tree or current branch in any form.
+
+We can try it on our current repository. Use this form of reset to return to what `09-Blame/readme.md` looked like before it was added:
```

Comment:
Do not use this GIT repository, create a special training repository for git-reset